src/screens/Reports/Report.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { Typography, List, ListItem, ListItemText } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { RouteComponentProps } from 'react-router-dom';
+import { selectCurrentSiteInformation, round } from '../../lib/redux/siteData';
+import { Person as PersonIcon, ShoppingCart as ShoppingCartIcon, Build as BuildIcon, Warning as WarningIcon } from '@material-ui/icons';
+import ListItemWrapper from '../../components/ListItemWrapper';
+import { useSelector } from 'react-redux';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    section: {
+      padding: '10px 5px',
+    },
+    header: {
+      display: 'inline-flex',
+      alignItems: 'center',
+    },
+    list: {
+      borderLeft: `5px solid ${theme.palette.primary.main}`,
+      padding: '2px 8px',
+      margin: '4px',
+    },
+    icon: {
+      marginRight: '5px',
+    },
+  });
+
+interface ReportProps extends RouteComponentProps {
+  classes: { section: string; header: string; list: string; icon: string; };
+  location: any;
+}
+
+interface Info {
+  label: string;
+  value: string;
+}
```

Comment:
I'm not really in love with the one-use interface here since it seems like it's only applicable for the `getListContent` function (and won't be used outside that component / function). Its name is also really general and doesn't provide context (though I can't think of a better name off hand). 

We could accomplish the same thing by just adding a comment line to `getListContent` explaining what the caller should pass to it.

Reply:
I want to keep it in to avoid any type - is there any way around this?